core/src/test/java/org/elasticsearch/bwcompat/BasicAnalysisBackwardCompatibilityIT.java

```diff
@@ -104,17 +104,8 @@ public void testAnalyzerTokensAfterUpgrade() throws IOException, ExecutionExcept
     }
 
     private String randomAnalyzer() {
-        while(true) {
-            PreBuiltAnalyzers preBuiltAnalyzers = RandomPicks.randomFrom(getRandom(), PreBuiltAnalyzers.values());
-            if (preBuiltAnalyzers == PreBuiltAnalyzers.SORANI && compatibilityVersion().before(Version.V_1_3_0)) {
-                continue; // SORANI was added in 1.3.0
-            }
-            if (preBuiltAnalyzers == PreBuiltAnalyzers.LITHUANIAN && compatibilityVersion().before(Version.V_2_1_0)) {
-                continue; // LITHUANIAN was added in 2.1.0
```

Comment:
I don't think it is since we don't support running clusters with mixed major versions